packages/core/strapi/lib/middlewares/ip.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+const ip = require('koa-ip');
+
+/**
+ * @type {import('./').MiddlewareFactory}
+ */
+module.exports = options => ip(options);
```

Comment:
Same as for compress + here we don't even modify options

Reply:
same answer I'm happy to do it too